2.0 tribe post (#139)
conan-io/conan-io.github.io: _posts/2020-07-28-Launching-Conan-2.0-Tribe.markdown

The feedback that we have received, especially contributions, bug reports, feature requests in Github has been spectacular. There are many users doing things with Conan that we couldn’t imagine, and that information is critical to keep improving, as those use cases and lessons learnt are something that we cannot do by ourselves. But sometimes, the conversations and discussions are a bit too local. For the next major leap, we need very solid and good quality feedback, in a consistent and unbiased way, over extended periods of time. This is the aim of the Conan 2.0 Tribe.

## The Conan 2.0 Tribe

The Conan 2.0 Tribe is a technical group of experienced and active Conan users and contributors, willing to provide quick and consistent feedback for defining the next major Conan 2.0 version over an extended period of time (several months). The tribe will kick off the 1st of September 2020, and first Conan 2.0-alpha releases are expected end of 2020

> The tribe will be always fully remote and distributed. The purpose of this tribe is to give feedback and to answer questions that will be of critical importance in the new design of Conan 2.0 (see the [Github milestone](https://github.com/conan-io/conan/milestone/59)). There are many things that there is already enough consistent feedback to take action, but there are still many open questions which would need an answer *before* starting working on them. A few examples:

Reviewer: ```suggestion
The tribe will be always fully remote and distributed. The purpose of this tribe is to give feedback and to answer questions that will be of critical importance in the new design of Conan 2.0 (see the [Github milestone](https://github.com/conan-io/conan/milestone/59)). There are many things with enough consistent feedback to take action already, but there are still many open questions that would need an answer *before* starting working on them. A few examples:
```

- What should be the default ``package_id_mode`` in Conan 2.0?
- Would it be ok if build-requirements were always resolved and be part of the graph?
- Should build-requirements be able to affect the ``package_id``?
- Which is the minimum CMake version that we should support? And the minimum Python version?
- Can multi-config (``self.cpp_info.debug`` and ``self.cpp_info.release``) packages be deprecated?

> These questions don’t have a "correct" answer, and the only way to do the right steps is to get good quality feedback, that is somewhat statistically representative of the Conan users base, and get it in relatively short periods of time.

Reviewer: ```suggestion
These questions don’t have a "correct" answer. The only way to do the right steps is to get statistically representative and good quality feedback from Conan users base in relatively short periods.
```
